Sources/Build/BuildOperation.swift

```diff
@@ -29,9 +29,12 @@ public final class BuildOperation: PackageStructureDelegate, SPMBuildCore.BuildS
     /// The closure for loading the package graph.
     let packageGraphLoader: () throws -> PackageGraph
     
-    /// The closure for invoking build tool plugins in the package graph.
-    let buildToolPluginInvoker: (PackageGraph) throws -> [ResolvedTarget: [BuildToolPluginInvocationResult]]
```

Comment:
I'm particularly happy about getting rid of this.  Although it mirrored the `packageGraphLoader`, it always felt kind of dodgy.